curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice.md

```diff
@@ -9,8 +9,13 @@ forumTopicId: 301509
 
 <section id='description'>
 Build a full stack JavaScript app that is functionally similar to this: <a href='https://url-shortener-microservice.freecodecamp.rocks/' target='_blank'>https://url-shortener-microservice.freecodecamp.rocks/</a>.
-Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.
-Start this project on Repl.it using <a href='https://repl.it/github/freeCodeCamp/boilerplate-project-urlshortener' target='_blank'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-urlshortener/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+Working on this project will involve you writing your code using one of the following methods:
```

Comment:
For consistency, I would have `Working on this...` be on the same line as `...rocks/</a>.` before it

Reply:
Can you make the suggestion here on GitHub, so I can commit it or is that not possible?

Reply:
It looks like that's not possible - GitHub disables the suggestion feature if the selection includes deleted lines.